mount.go

```diff
@@ -229,6 +230,26 @@ func initFuseFrontend(key []byte, args *argContainer, confFile *configfile.ConfF
 		HKDF:           args.hkdf,
 		SerializeReads: args.serialize_reads,
 		ForceDecode:    args.forcedecode,
+		ForceOwner:     nil,
+	}
+	if args.force_owner != "" {
+		ownerPieces := strings.SplitN(args.force_owner, ":", 2)
+		if len(ownerPieces) != 2 {
+			tlog.Fatal.Printf("force_owner must be in form UID:GID")
+			os.Exit(exitcodes.Usage)
+		}
+		uidNum, err := strconv.ParseInt(ownerPieces[0], 0, 32)
+		if err != nil || uidNum < 0 {
+			tlog.Fatal.Printf("force_owner: Unable to parse UID %v as positive integer", ownerPieces[0])
+			os.Exit(exitcodes.Usage)
+		}
+		gidNum, err := strconv.ParseInt(ownerPieces[1], 0, 32)
+		if err != nil || gidNum < 0 {
+			tlog.Fatal.Printf("force_owner: Unable to parse GID %v as positive integer", ownerPieces[1])
+			os.Exit(exitcodes.Usage)
+		}
+		frontendArgs.ForceOwner = &fuse.Owner{Uid: uint32(uidNum), Gid: uint32(gidNum)}
+		args.allow_other = true
```

Comment:
Parsing looks good but should be moved to main. The problem with doing it here is that we may abort with a parse error **after** the user has entered the password. I think we should do all validation **before** asking the user to do work (here: prompt for the password) because that work is wasted when we abort.

I suggest you add a helper variable to argContainer like I have done with `_ctlsockFd`, for example named `_ForceOwner`. Then you can parse the force_owner string in main and store the result in _ForceOwner.